stratum/hal/lib/common/hal.cc

```diff
@@ -32,7 +32,7 @@
 #include "absl/synchronization/mutex.h"
 
 // TODO(unknown): Use FLAG_DEFINE for all flags.
-DEFINE_string(external_stratum_urls, "",
+DEFINE_string(external_stratum_urls, stratum::kExternalStratumUrls,
```

Comment:
If we set a default here, we could remove all `--external_stratum_urls`` mentions from flag files, start script, etc.